db/migrate/20150220194753_add_user_id_to_reservations.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserIdToReservations < ActiveRecord::Migration
+  def change
+    rename_column :reservations, :guest_id, :user_id
+    remove_column :reservations, :host_id
+  end
+end
```

Comment:
Awesome! now we can get my trips and my reservation links going in the profiel dropdown! :+1: